Neos.Flow/Classes/Session/Data/SessionMetaDataStore.php

```diff
@@ -14,27 +14,25 @@
  */
 
 use Neos\Cache\Backend\IterableBackendInterface;
+use Neos\Cache\Exception;
 use Neos\Cache\Exception\InvalidBackendException;
+use Neos\Cache\Exception\NotSupportedByBackendException;
 use Neos\Cache\Frontend\VariableFrontend;
-use Neos\FLow\Annotations as Flow;
+use Neos\Flow\Annotations\InjectConfiguration;
 use Neos\Flow\Session\Exception\InvalidDataInSessionDataStoreException;
 
 /**
  * @internal
  */
 class SessionMetaDataStore
 {
-    protected VariableFrontend $cache;
-
     protected const TAG_PREFIX = 'customtag-';
+    protected const GARBAGE_COLLECTION_CACHE_IDENTIFIER = '_garbage-collection-running';
```

Comment:
Not against it, but I am pretty sure this was a deliberate naming choice:

- it is a "garbage-collection"-cache-identifier,
- not a "garbage-collection-cache"-identifier

🤣

Reply:
I chose that name and I am sure I did not put that much thought into it :-)